packages/bodiless-components/src/withBodilessLinkToggle.tsx

```diff
@@ -12,25 +12,17 @@
  * limitations under the License.
  */
 
-import { Fragment } from 'react';
-import {
-  withSidecarNodes,
-  ifReadOnly, ifEditable, withOnlyProps,
-} from '@bodiless/core';
+import { withSidecarNodes } from '@bodiless/core';
 import { flowRight, identity } from 'lodash';
 import { replaceWith, withoutProps, withDesign } from '@bodiless/fclasses';
 import type { HOC } from '@bodiless/fclasses';
 import { withChameleonComponentFormControls, applyChameleon, withChameleonContext } from './Chameleon';
 
-const SafeFragment = withOnlyProps('key', 'children')(Fragment);
-const Span = withoutProps('')('span');
+const Div = withoutProps('')('div');
 
 const withBodilessLinkToggle = (asEditableLink: HOC) => flowRight(
   withDesign({
-    _default: flowRight(
-      ifEditable(replaceWith(Span)),
-      ifReadOnly(replaceWith(SafeFragment)),
-    ),
+    _default: replaceWith(Div),
     Link: identity,
```

Comment:
We keep link styles unchanged -- meaning if there is a link component in the menu it will render `<Title ...>` as an `<a ..>` on both: preview and edit. By default `<Title ..>` component is just a `<div>` for both: preview and edit envs. This way we don't lose `<Title ..>` styles in preview.

Reply:
So .. I"m guessing you had no luck with applying those styles to the component  (the editor)  itself?
At any rate - I don't think we should default this to a Div in withBodilessLInkToggle - I think we should make use of the  _default design key at site level to replace it with a div there.

Reply:
Updated